lib/preload.ts

```diff
@@ -587,16 +609,32 @@ export class Preloader extends EventEmitter {
 					ci.image[0].is_stored_at__image_location.toLowerCase(),
 			});
 		});
+
 		// App from the state endpoint (v1 or v2 depending on the supervisor version).
-		const app = _.values(this.state.local.apps)[0];
 		if (this._supervisorLT7()) {
+			const app = _.values(this.state.local.apps)[0];
```

Comment:
Should we adjust `_getAppServices` to support all versions and use it everywhere for consistency?

Reply:
Yeah, that would simplify the code. I'll make the change